tests/build/build.go

```diff
@@ -83,6 +85,9 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 			Expect(err).ShouldNot(HaveOccurred())
 
 			defaultBranchTestComponentName = fmt.Sprintf("test-custom-default-branch-%s", util.GenerateRandomString(4))
+			annotationsComponentName = fmt.Sprintf("test-simple-build-%s", util.GenerateRandomString(4))
```

Comment:
nit: `test-simple-build-%s` naming is a bit confusing, since we are using this component name to trigger advance build as well

Reply:
Okay, I propose change to `test-annotations-%s`. What do you think?